tests/integration_tests/functional/test_balloon.py

```diff
@@ -205,7 +202,7 @@ def test_deflate_on_oom(uvm_plain_any, deflate_on_oom):
 
     # We get an initial reading of the RSS, then calculate the amount
     # we need to inflate the balloon with by subtracting it from the
-    # VM size and adding an offset of 10 MiB in order to make sure we
+    # VM size and adding an offset of 20 MiB in order to make sure we
```

Comment:
looks like the offset is still 10M

Reply:
Ah, yeah, fixed. Originally my approach for fixing it was increasing this to 20, forgot to revert the comment change :)